ibverbs/src/lib.rs

```diff
@@ -74,6 +74,7 @@ use std::os::raw::c_void;
 use std::ptr;
 
 const PORT_NUM: u8 = 1;
+const MAX_GID_TBL_LEN: usize = 32;
```

Comment:
Is this const defined in such a way that we can get it out of `ibverbs-sys` instead somehow?